dev.hdos.HDOS.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>dev.hdos.HDOS</id>
+    <name>HDOS</name>
+    <developer_name>HDOS</developer_name>
+    <summary>HDOS OSRS Client</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
```

Comment:
```suggestion
    <project_license>LicenseRef-proprietary=https://github.com/flathub/dev.hdos.HDOS/LICENSE</project_license>
```

You can change it to something along this **after** the app is merged to Flathub,